.eslintrc.json

```diff
@@ -103,40 +103,67 @@
         "patterns": [
           // Restrict all relative imports except for .css files.
           {
-            "group": ["./*", "../*", "!./*.css", "!../*.css"],
+            "group": [
```

Comment:
https://github.com/KittyCAD/modeling-app/blob/main/biome.json#L46

Well we have explicitly turned of biome for json files. This appears to be the formatter from my VSCode. 

The first line of the `npm run fmt` says `biome format --write .eslintrc.json` which is totally wrong because we ignore all json files.